logviewer/build.gradle

```diff
@@ -36,35 +36,47 @@ jooq {
 }
 
 dependencies {
-    compileOnly 'org.apache.logging.log4j:log4j-api:2.15.0'
-    runtimeOnly 'org.apache.logging.log4j:log4j-jul:2.15.0'
-    runtimeOnly 'org.apache.logging.log4j:log4j-slf4j-impl:2.15.0'
+    implementation('org.apache.logging.log4j:log4j-api') {
+        version {
+            require '2.16.0'
+        }
+        because 'Log4Shell happened'
+    }
+    runtimeOnly('org.apache.logging.log4j:log4j-core') {
+        version {
+            require '2.16.0'
+        }
+        because 'Log4Shell happened'
+    }
```

Comment:
```suggestion
    implementation('org.apache.logging.log4j:log4j-api') {
        version {
            require '2.16.0'
        }
        because 'Log4Shell happened'
    }
    
    runtimeOnly('org.apache.logging.log4j:log4j-core') {
        version {
            require '2.16.0'
        }
        because 'Log4Shell happened'
    }
```

Reply:
Add gap to make the readability easer if you want.

Reply:
I say no, its better to group by topics (log4j) and have empty lines when the topic changes.